ogr/ogrsf_frmts/miramon/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+add_gdal_driver(
+  TARGET ogr_MiraMon
+  SOURCES ogrmiramondatasource.cpp ogrmiramondriver.cpp ogrmiramonlayer.cpp mm_wrlayr.c mm_gdal_functions.c mm_rdlayr.c
+  BUILTIN)
```

This should be PLUGIN_CAPABLE, so that it is possible to disable its building (BUILTIN is only for core drivers that for some reason cannot be disabled. generally because another driver depend on them, or some core functionality depend on them)

```suggestion
  PLUGIN_CAPABLE)
```